### PR TITLE
Use $__interval  for resources dashboards

### PR DESCRIPTION
**Pull request by povilasv**
Ref https://github.com/kubernetes-monitoring/kubernetes-mixin/issues/353

### CODE REVIEW

File: dashboards/resources.libsonnet [hunk 1]
```diff
@@ -5,31 +5,6 @@ local template = grafana.template;
 {
   grafanaDashboards+:: {
 
-    local intervalTemplate =
-      template.new(
-        name='interval',
```
Comment (brancz) [inline dashboards/resources.libsonnet]: I would say let’s not remove this but set the value to $__interval

**Comment by povilasv** [comment]
Fixed it, but accidentally also formatted jsonnet :/

File: dashboards/resources.libsonnet (hunk 1 above)
Comment (povilasv) [reply dashboards/resources.libsonnet]: :+1:

**Comment by paulfantom** [comment]
lgtm, but could you split commits containing formatting fix from one with `$__interval` fix to allow easier review?

**Comment by povilasv** [comment]
Made a PR https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/363

**Comment by brancz** [comment]
Merged the other one let’s rebase this.

File: dashboards/resources/node.libsonnet [hunk 2]
```diff
@@ -23,8 +23,9 @@ local template = grafana.template;
         options: [
           {
             selected: true,
-            text: '4h',
-            value: '4h',
+            text: '$__interval',
+            value: '$__interval',
+>>>>>>> Fix
```
Comment (brancz) [inline dashboards/resources/node.libsonnet]: Rebase leftover

File: dashboards/resources/cluster.libsonnet [hunk 3]
```diff
@@ -25,6 +25,8 @@ local template = grafana.template;
             selected: true,
             text: '4h',
             value: '4h',
+            text: '$__interval',
```
Comment (brancz) [inline dashboards/resources/cluster.libsonnet]: text and value field are duplicate here

**Comment by brancz** [comment]
Awesome :100: thanks!

**Pull request merged**